at_secondary/at_persistence_secondary_server/lib/src/notification/at_notification_keystore.dart

```diff
@@ -134,6 +135,39 @@ class AtNotificationKeystore
     return expiredKeys;
   }
 
+  Future<void> checkExpiry() async {
```

Comment:
* move the logic to getExpiredKeys..
* default the number of hrs to 72..move this value to config.yaml

Reply:
Changes have been made as suggested.